### PR TITLE
Prepare for publication to crates.io

### PR DESCRIPTION
**Pull request by SimonSapin**
https://github.com/apollographql/router/issues/491

### CODE REVIEW

File: apollo-router/testing_schema.graphql [hunk 1]
```diff
@@ -0,0 +1,73 @@
+schema
```
Comment (SimonSapin) [inline apollo-router/testing_schema.graphql:1]: This file is a copy of `examples/graphql/local.graphql`, no content change.

**Review by abernix — APPROVED** [review]

File: apollo-router/build/spaceport.rs [hunk 2]
```diff
@@ -23,30 +20,16 @@ pub fn main() -> Result<(), Box<dyn Error>> {
...
-        copy(&mut content.as_bytes(), &mut dest)?;
-    }
+    std::fs::write(&reports, &content)?;
 
     // Process the proto files
-    let proto_files = vec![
-        proto_dir.join("agents.proto"),
-        proto_dir.join("reports.proto"),
-    ];
-
```
Comment (garypen) [inline apollo-router/build/spaceport.rs:49]: Ah, now I remember why I couldn't put it into OUT_DIR. This was a little trick to try and avoid annoying re-compilations when nothing has really changed. Is there a way to preserve this or is it not actually required now?

Comment (SimonSapin) [reply apollo-router/build/spaceport.rs]: Indeed, this PR caused a rebuild every time. I’ve removed `cargo:rerun-if-changed` for `reports.proto` to fix it.

It means we won’t automatically recompile if https://usage-reporting.api.apollographql.com/proto/reports.proto has changed, but that was effectively already the case.

**Review by garypen — APPROVED** [review]

File: apollo-router/build/spaceport.rs (hunk 2 above)
Comment (SimonSapin) [reply apollo-router/build/spaceport.rs]: From discussion with Gary: I’ll make a follow-up PR to add `reports.proto` to the source repository, have the build script use that without downloading, add an xtask command to download and update the file, and add a test that downloads and checks our copy is up to date.

**Pull request merged**